components/Account/index.js

```diff
@@ -226,16 +224,16 @@ export default compose(
         hasActiveMemberships,
         memberships: hasMemberships && data.me.memberships,
         hasAccessGrants,
-        recurringAmount: hasMemberships
-          ? max(
-              data.me.memberships.map(m => {
-                const recurringOptions = m.pledge.options.filter(
-                  o => o.reward && o.reward.name === 'MONTHLY_ABO'
-                )
-                return max(recurringOptions.map(o => o.price)) || 0
-              })
-            )
-          : 0
+        isAutoPaying:
+          hasMemberships &&
+          data.me.memberships.some(m => {
+            if (!m.active || !m.renew) {
+              return false
+            }
+            if (m.type.name === 'MONTHLY_ABO' || m.autoPay) {
+              return true
+            }
```

Comment:
Assuming it has to be active and renewable, either as a MONTHLY_ABO or with auto-pay-flag set to true, then condition could be:

```suggestion
            return m.active && m.renew && (m.type.name === 'MONTHLY_ABO' || m.autoPay)
```

(Changes untested.)